sdk/java-sdk/sdk-reactor/src/main/java/spec/sdk/reactor/v1/domain/enhanced/MetricsRuntimes.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 Layotto Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package spec.sdk.reactor.v1.domain.enhanced;
+
+/**
+ * Metrics Runtimes standard API defined.
+ */
+public interface MetricsRuntimes {
```

Comment:
Is there an API definition for application layer observability?

We used `opentelemetry` to make a version of [TelemetryAPI](https://github.com/reactivegroup/cloud-runtimes-jvm/blob/master/cloud-runtimes-api/src/main/java/group/rxcloud/cloudruntimes/domain/enhanced/TelemetryRuntimes.java), which is used for business code to actively initiate metrics/traces.

Reply:
Currently no,but of course we can discuss whether and how to add it (in a new issue ?)

Reply:
yes